### DISCUSSION TITLE
Aux sensitivity and hold time settings

### DISCUSSION THREAD

**Discussion by MarkFSI** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#154] [Editor App Feature Requests]
Briefly, I've noticed that the settings of sensitivity and hold time for aux switches are not available in the editor app, but only through the device's interface. So I suggest to add they in the app ;-)

**Comment by TheGuacamoleXplosion** [comment]
I didn't even know those settings existed on the device! I connected two different switches last weekend, set to bank up/down, and had lots of trouble with unwanted double-triggering (hitting switch once would advance 2 banks instead of 1). Happened with both switches. Have to retry with those sensitivity/hold time settings!

**Reply by TheGuacamoleXplosion** [reply]
Tried it ... unfortunately, still getting unwanted double-triggering; in Low, Med and High sensitivity modes, with two different footswitches. In low mode, double-triggering (2 banks up/down at one kick instead of 1) happens less often, but in exchange, some kicks don't trigger any action.
